projects/jdwp/runtime/async_streams.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+
+
+class AsyncInputStream:
```

Comment:
Let's only define the interface for now, we'll deal with actual implementation in Michael's PRs.

1. Make this abstract (`class JDWPInputStreamBase(abc.ABC):`)
2. Remove existing methods
3. For each of [these types](https://github.com/facebookexperimental/ExtendedAndroidTools/blob/main/projects/jdwp/defs/schema.py#L26-L62) define an asynchronous abstract method that reads a single value of that type. Do not bother with the types that are commented out.